src/mongo_client.ts

```diff
@@ -252,6 +252,11 @@ export interface MongoClientOptions extends BSONSerializeOptions, SupportedNodeC
   srvPoller?: SrvPoller;
   /** @internal */
   connectionType?: typeof Connection;
+  /**
+   * @internal
+   * TODO: NODE-5671 - remove internal flag
+   */
```

Comment:
This and and mongo_client.test.js changes are from NODE-4849, which this branch branches off of. I don't know how to have it only show the NODE-5452 changes.